### PR TITLE
[script.user.rating] 1.0.0

### PR DESCRIPTION
**Pull request by Regss**

### CODE REVIEW

File: script.user.rating/addon.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.user.rating"
+    name="User Rating"
+    version="1.0.0"
+    provider-name="Regss">
+    <requires>
+        <import addon="xbmc.python" version="2.24.0" />
+        <import addon="service.user.rating" version="1.0.0" />
```
Comment (tamland) [inline script.user.rating/addon.xml]: Why do you require this addon?

**Comment by tamland** [comment]
I'd like you to provide a better description of this addon in addon.xml as well. It is not clear what it is and does other than that it has something to do with rating.

**Comment by Regss** [comment]
It is service for this script. When video stop playing or user change watched status it run this script. [https://github.com/Regss/service.user.rating](url) I create PR for this service after this script will be merged.

**Comment by tamland** [comment]
You need to do it the other way around. I cannot merge this without all dependencies being fulfilled.

Ps: if your addons do not work standalone, consider using multiple extension point in one addon

**Comment by Regss** [comment]
I add extension point and and more description. It is now enough? Do I need to change anything else?

File: script.user.rating/addon.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.user.rating"
+    name="User Rating"
+    version="1.0.0"
+    provider-name="Regss">
+    <requires>
+        <import addon="xbmc.python" version="2.24.0" />
+    </requires>
+    <extension point="xbmc.python.script" library="default.py" />
+    <extension point="xbmc.service" library="service.py" start="[login]" />
```
Comment (tamland) [inline script.user.rating/addon.xml]: should be `start="login" />`

**Comment by Regss** [comment]
fixed

**Pull request merged**